pyproject.toml

```diff
@@ -16,6 +17,9 @@ dependencies = [
   "xxhash>=3.5.0",
 ]
 
+[tool.hatch.metadata]
+allow-direct-references = true
```

Comment:
I think that you don't need this when using `tool.uv.sources` like for `ethereum`. please remove the `@ git+` in the dep, remove this and update the source section